File: ARCADE/The Core/Level 2_1: reachNextLevel.cs
    using System;
     
    public class Test
    {
    	public static bool reachNextLevel(int experience, int threshold, int reward){
            return (experience + reward >= threshold);
        }
    	public static void Main()
    	{
    		int experience = 10, threshold = 15, reward = 5;
    		Console.WriteLine(reachNextLevel(experience, threshold, reward));
    		int experience1 = 10, threshold1 = 15, reward1 = 4;
    		Console.WriteLine(reachNextLevel(experience1, threshold1, reward1));
    	}
    }

File: ARCADE/The Core/Level 2_1: reachNextLevel.pas
program reachNextLevel;

function reachNextLevel(experience: integer; threshold: integer; reward: integer): boolean;
begin
    if(experience + reward >= threshold) then reachNextLevel := true
    else reachNextLevel := false;
end;

var experience, threshold, reward : integer;
begin
	experience := 10; threshold := 15; reward := 5;
	writeln(reachNextLevel(experience, threshold, reward));
	experience := 10; threshold := 15; reward := 4;
	writeln(reachNextLevel(experience, threshold, reward));
end.
